gwadoc/doc_en.py

```diff
@@ -816,11 +819,28 @@
 a word that names a part of a larger whole
 """
 relations.meronym.dfn.en = """
-A relation between two concepts where concept Y makes up a part of
-concept X.
+A relation between two concepts where concept A makes up a part of
+concept B.
 """
 relations.meronym.ex.en = "hand/finger"
 relations.meronym.exe.en = """
+ * *<sense: pwn-3.0:04574999-n:wheel>* ``meronym`` *<sense: pwn-3.0:02958343-n:automobile>*
+ * *<sense: pwn-3.0:03138534-n:crown>* ``meronym`` *<sense: pwn-3.0:03497657-n:hat>*
+ * *<sense: pwn-3.0:05217168-n:human body>* ``meronym`` *<sense: pwn-3.0:00007846-n:person>*
```

Comment:
For these I think we will do our own substitution, so they can be changed to a form like this:

    * `wheel <ILIURL/61096>`_ is a ``meronym`` of `wheeled vehicle <ILIURL/61100>`_

Where `ILIURL` is that exact string and it will get replaced during the build process.

Reply:
I will effect the changes. Thanks